src/main/kernel/Manager/BundleManager.php

```diff
@@ -55,7 +55,7 @@ public function getActiveBundles($fetchAll = false)
         $configProviderBundles = [];
         $nonAutoConfigurableBundles = [];
         $environment = $this->getEnvironment();
-        $updateMode = 'cli' === PHP_SAPI;
+        $updateMode = file_exists($this->kernel->getProjectDir().'/files/config/.update');
```

Comment:
It should be `MaintenanceHandler::isMaintenanceEnabled()`.
IMHO, it is a little less ugly if you move it here : https://github.com/claroline/Claroline/blob/5b2278fdb0a183ed873f2c27e14de03a24f6525f/src/main/kernel/ClarolineKernelBundle.php#L47 and here : https://github.com/claroline/Claroline/blob/5b2278fdb0a183ed873f2c27e14de03a24f6525f/src/main/kernel/ClarolineKernelBundle.php#L60

And simply remove this check.